server/game/basecard.js

```diff
@@ -443,6 +443,15 @@ class BaseCard {
         return false;
     }
 
+    getShortSummary() {
```

Comment:
If you have a better name for this, please suggest it. It's being used for card preview on the client side, but didn't want to tie the naming to that usage.